specification/cognitiveservices/data-plane/CustomSearch/v1.0/CustomSearch.json

```diff
@@ -0,0 +1,598 @@
+			"allOf": [{
+				"$ref": "#/definitions/Response"
+			}],
+			"type": "object",
+			"required": [
+				"errors"
+			],
+			"properties": {
+				"errors": {
+					"description": "A list of errors that describe the reasons why the request failed.",
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/Error"
+					}
+				}
+			}
+		},
+		"Answer": {
+			"allOf": [{
+				"$ref": "#/definitions/Response"
+			}],
+			"type": "object",
+			"properties": {
+				"followUpQueries": {
+					"readOnly": true,
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/Query"
+					}
+				}
+			}
+		},
+		"Query": {
+			"description": "Defines a search query.",
+			"type": "object",
+			"required": [
+				"text"
+			],
+			"properties": {
+				"text": {
+					"description": "The query string. Use this string as the query term in a new search request.",
+					"type": "string"
+				},
+				"displayText": {
+					"description": "The display version of the query term. This version of the query term may contain special characters that highlight the search term found in the query string. The string contains the highlighting characters only if the query enabled hit highlighting",
+					"readOnly": true,
+					"type": "string"
+				},
+				"webSearchUrl": {
+					"description": "The URL that takes the user to the Bing search results page for the query.Only related search results include this field.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"searchLink": {
+					"readOnly": true,
+					"type": "string"
+				}
+			}
+		},
+		"Thing": {
+			"allOf": [{
+				"$ref": "#/definitions/Response"
+			}],
+			"type": "object",
+			"properties": {
+				"name": {
+					"description": "The name of the thing represented by this object.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"url": {
+					"description": "The URL to get more information about the thing represented by this object.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"description": {
+					"description": "A short description of the item.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"bingId": {
+					"description": "An ID that uniquely identifies this item.",
+					"readOnly": true,
+					"type": "string"
+				}
+			}
+		},
+		"CreativeWork": {
+			"allOf": [{
+				"$ref": "#/definitions/Thing"
+			}],
+			"type": "object",
+			"properties": {
+				"thumbnailUrl": {
+					"description": "The URL to a thumbnail of the item.",
+					"readOnly": true,
+					"type": "string"
+				},
+				"provider": {
+					"description": "The source of the creative work.",
+					"readOnly": true,
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/Thing"
+					}
+				},
+				"text": {
+					"readOnly": true,
+					"type": "string"
+				}
+			}
+		},
+		"WebMetaTag": {
+			"description": "Defines a webpage's metadata.",
+			"x-ms-discriminator-value": "Web/MetaTag",
+			"type": "object",
```

Comment:
Which is the discriminator property here?

Reply:
What is a discriminator property?

Reply:
It should be _type
 "_type": {
          "type": "string"
        }

Based on https://github.com/Azure/oav/pull/171, We don't need to specify discriminator property for every child model.

Reply:
Is this a base type?

Reply:
This is a derived type

Reply:
But it has no base property, why is the discriminator value needed?

Reply:
Will remove
@q3blend : Could you please remove "x-ms-discriminator-value": "Web/MetaTag" as it truly not needed now

Reply:
Removed